src/Compilers/Core/Portable/Compilation/Expression.cs

```diff
@@ -104,6 +104,8 @@ public ConditionalChoice(IExpression condition, IExpression ifTrue, IExpression
 
         public OperationKind Kind => OperationKind.ConditionalChoiceExpression;
 
+        public bool HasErrors => false;
```

Comment:
Could we not have a common base type that these all get this functionality from?